move/interchain_token_service/sources/versioned/interchain_token_service_v0.move

```diff
@@ -175,19 +175,29 @@ module interchain_token_service::interchain_token_service_v0 {
         &self.version_control
     }
 
+    /// Originally, the versioned function for public fun interchain_token_service::register_coin.
+    /// Now, a worker function to process registering a coin (modified in version 1.0).
+    /// @see self.register_coin_from_info
+    /// @see self.register_coin_from_metadata
```

Comment:
```suggestion
```
Not too useful I think, should be clear by following references from the code itself. Ideally, comments should only be added if the code can't be made self-descriptive, since they add maintenance overhead